blueocean-core-js/src/js/jwt.js

```diff
@@ -41,7 +41,8 @@ export default {
      */
     verifyToken(token, certObject) {
         return new Promise((resolve, reject) =>
-            jwt.verify(token, jwk2pem(certObject), { algorithms: [certObject.alg] }, (err, payload) => {
+            // Allow the clocks to be 60s out of sync.
+            jwt.verify(token, jwk2pem(certObject), { algorithms: [certObject.alg], clockTolerance: 60 }, (err, payload) => {
```

Comment:
This change actually has nothing to do with IE. Please read below.

@imeredith @vivek please check this out ... look at commit https://github.com/jenkinsci/blueocean-plugin/pull/447/commits/4b41be19a26983057ddef3fffc4c58594efd58fc

Without this change (60s +/- is something that can be debated), [`jsonwebtoken.verify`](https://github.com/auth0/node-jsonwebtoken/blob/master/verify.js#L111-L113) throws a `NotBeforeError` if the clock times on the client and server are out of sync by more than 0s (rounded).

I set the tolerance to 60s just to test - I didn't see anything on the `certObject` that could be used. Seems like the tolerance should be something that comes with the token, or something else on the server.